src/components/FormProduct/index.jsx

```diff
@@ -69,7 +71,7 @@ export const FormProduct = () => {
             type="radio"
             required="required"
             name="nextPurchase"
-            value={7}
+            value={6}
```

Comment:
I was wondering why is it necessary to change values to `nextPurchase`, is there a reason? Maybe I'm wrong, but I remember in a later issue #4 there were other values: 7, 14, and 30. Please, could you help me with this doubt? I'll appreciate it.

Reply:
If in issue 4 it says an approximate ... Then to meet the acceptance criteria of this issue a quick solution is to change the previous nextPuchase and I continue to meet both issues